### PR TITLE
Update goentri.com.clickmax.json

### PR DESCRIPTION
**Pull request by evelyndearaujo-goentri**

### CODE REVIEW

File: goentri.com.clickmax.json [hunk 1]
```diff
@@ -17,6 +17,11 @@
       "host": "@",
       "pointsTo": "%cnamePointsTo%",
       "ttl": 3600
+    },
+    {
+      "type": "REDIR301",
+      "target": "%target%",
+      "host": "@"
```
Comment (pawel-kow) [inline goentri.com.clickmax.json:24]: no a chance this template would work. CNAME cannot co-exist with A/AAAA records needed for the redirect on the same host.

**Review by pawel-kow — CHANGES_REQUESTED** [review]

File: goentri.com.clickmax.json (hunk 1 above)
Comment (evelyndearaujo-goentri) [reply goentri.com.clickmax.json:24]: Would this work if the host required is set to false and the CNAME host is back to www?

Comment (pawel-kow) [reply goentri.com.clickmax.json:24]: yes, if you place CNAME on "www" then host required can be set to false

Comment (evelyndearaujo-goentri) [reply goentri.com.clickmax.json:24]: it's done.

**Review by pawel-kow — APPROVED** [review]
lgtm

**Pull request merged**